packages/docs/docs/00-introduction.mdx

```diff
@@ -118,9 +118,32 @@ Apache 2
 
 ## Credits
 
-| ![mozilla-services-logo](https://avatars1.githubusercontent.com/u/1066228?s=200&v=4)                 | ![browserstack logo](https://user-images.githubusercontent.com/1689183/51487090-4ea04f80-1d57-11e9-9a91-79b7ef8d2013.png) | ![netlify logo](https://www.netlify.com/img/global/badges/netlify-color-accent.svg) |
-| ---------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------- |
-| This project initially started as a [mozilla-services](https://github.com/mozilla-services) project. | Testing is powered by [BrowserStack](https://www.browserstack.com/).                                                      | Deploy Previews are provided by [Netlify](https://www.netlify.com).                 |
+<table>
+    <tr>
+        <th>
+            <img src="https://avatars1.githubusercontent.com/u/1066228?s=200&v=4" alt="mozilla-services-logo" style={{maxHeight: '100px'}} />
+        </th>
+        <td>
+            This project initially started as a <a href="https://github.com/mozilla-services">mozilla-services</a> project.
+        </td>
+    </tr>
+    <tr>
+        <th>
+            <img src="https://user-images.githubusercontent.com/1689183/51487090-4ea04f80-1d57-11e9-9a91-79b7ef8d2013.png" alt='browserstack logo'  style={{maxHeight: '100px'}}/>
+        </th>
+        <td>
+            Testing is powered by <a href="https://www.browserstack.com/" >BrowserStack</a>.
+        </td>
+    </tr>
+    <tr>
+        <th>
+            <img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" alt='netlify logo'  style={{maxHeight: '100px'}}/>
+        </th>
+        <td>
+            Deploy Previews are provided by <a href="https://www.netlify.com">Netlify</a>.
+        </td>
+    </tr>
+</table>
```

Comment:
Just flipped the orientation of this table and added a max height to the images so this looks a little cleaner (even though we aren't using BrowserStack or Netlify currently): 

![image](https://github.com/rjsf-team/react-jsonschema-form/assets/17580037/1a559b6c-0e06-4e99-afdf-43c5cf28fa81)